apps/mobile/src/components/register-adoption-form/compose/form-steps/pet-upload-photos-step/pet-upload-photos-step.hooks.test.ts

```diff
@@ -0,0 +1,36 @@
+import { act, renderHook, waitFor } from '@testing-library/react-native';
+
+import { usePetPhotosPicker } from './pet-upload-photos-step.hooks';
+
+jest.mock('../../../../../shared/image-picker', () => ({
+  openImageLibrary: jest.fn(),
+}));
+
+const SLOTS = [undefined, 'image1', undefined];
+
+describe('usePetPhotosPicker', () => {
+  beforeEach(jest.restoreAllMocks);
+
+  it('returns initial images state and pickImage function', () => {
+    const { result } = renderHook(() => usePetPhotosPicker());
+
+    expect(result.current.images).toEqual([undefined, undefined, undefined]);
+    expect(typeof result.current.pickImage).toBe('function');
+  });
+
+  it.each(SLOTS)('pickImage updates the images state', async (slot) => {
```

Comment:
Jest output doesn't look clear 

<img width="513" alt="Screen Shot 2023-03-08 at 18 55 52" src="https://user-images.githubusercontent.com/19699724/223859578-56162b62-8b0b-4ef2-8da9-24f0147b463b.png">

Could you concatenate an index just so we can differentiate each one?